requirements-dev.txt

```diff
@@ -1,4 +1,15 @@
-codecov
-coverage
-tox
-unittest2
+certifi==2017.11.5
```

Comment:
Could we document somewhere the top level requirements  ? 

as a side note, for devel and doc, I would suggest to not pin. 

Or if we keep them pin, let's make that the require badge look at the files

Reply:
I think we should keep the dev requirements pinned since for testing we do 
```
pip install -r requirements.txt -r requirements-dev.txt
```
We want to reproduce the same environment over and over again. I agree with the requirements-doc.txt though. I think there is no reason to pin the doc requirements. Requires.io detects all the requirements files already(https://requires.io/github/NeurodataWithoutBorders/pynwb/requirements/?branch=add-requirements-doc).

Reply:
For now, let's move forward with this. We can revisit if it cause problem.